nodebuilder/das/config.go

```diff
@@ -0,0 +1,70 @@
+package das
+
+import (
+	"errors"
+	"time"
+)
+
+// Config contains configuration parameters for the DASer (or DASing process)
+type Config struct {
```

Comment:
@derrandz can you please export `params` and its fields from `das` package and make a 
`type Config das.Params`  ? Also, we don't technically need this `DefaultConfig()` since the `daser` is taking options rather than a config / config values.

Reply:
This is with the goal of deduplication of the code
For clarification: we need to keep the optional parameters approach.

Reply:
Yeah I see what you are saying. This was an attempt to be compliant with the pattern used with the other modules but it's very redundant.